caktus_primer/static/stylus/form.styl

```diff
@@ -83,10 +87,19 @@ fieldset
         pointer-events: none
         transition: all 0.2s
 
-    input.populated:not([type=checkbox]):not([type=radio]):not([type=file]) + label, input:focus:not([type=checkbox]):not([type=radio]):not([type=file]) + label, input[readonly] + label,input[disabled] + label, input[readonly] + label
+    input.populated:not([type=checkbox]):not([type=radio]):not([type=file]) + label,
+    input:focus:not([type=checkbox]):not([type=radio]):not([type=file]) + label,
+    input[readonly] + label,
+    input[disabled] + label
         graduated(font-size, $font-size-very-small, $font-size-small, 0)
         top: -3.5em
 
+    input:focus:not([type=checkbox]):not([type=radio]):not([type=file]) + label
+        color: $palette.accent.base
+
+    input.input-error:not([type=checkbox]):not([type=radio]):not([type=file]) + label
+        color: $palette.alert.base
```

Comment:
We don't wan to change the color of the label, and after the changes you aren't even using `<label>` in the search component markup

Reply:
So, I'd like to rope this change into this PR because I can.  And I know how these types of things get backlogged.  True, this is no longer relevant to the Search inputs, but I ran it by Basia and she liked the change.  It gives the user an indication that the input is focused.  For reference, material design does something similar: https://material.io/guidelines/components/text-fields.html#text-fields-states